scripts/generateESLintTest.js

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
I still find this really weird, and I've rarely seen stuff like this in the wild. If you are concerned with people not being able to see all the output in one place, why not just add `eslint .` as part of your `npm test`? I don't see how people can be frustrated waiting for CI to run linters and are unwilling to run a command locally that can tell them the exact ESLint output.